dev/com.ibm.ws.microprofile.faulttolerance_fat_tck/publish/tckRunner/tck/tck-suite.xml

```diff
@@ -0,0 +1,15 @@
+<!-- Licensed under the Apache License, Version 2.0 (the "License"); you
+        may not use this file except in compliance with the License. You may obtain
+        a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless
+        required by applicable law or agreed to in writing, software distributed
+        under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+        OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+        the specific language governing permissions and limitations under the License. -->
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="microprofile-fault-tolerance-TCK" verbose="2" configfailurepolicy="continue">
```

Comment:
why are there three different suite xml files?

Reply:
The are there as examples. It is not obvious how to do what each of them does (for example the beanshell material) so I purposefully curated the list of what I left in to provide some examples as this PR and the config one will be used as examples by other people.